integrations/influxdb_connector/influxdb-timestream-connector/README.md

```diff
@@ -47,30 +47,70 @@ The InfluxDB Timestream connector can be deployed within an AWS CloudFormation s
 
 #### Deploying a CloudFormation Stack Using SAM CLI
 
-The stack can be deployed using the [AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/install-sam-cli.html) and `template.yml`.
+The stack can be deployed using the [AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/using-sam-cli.html), [Cargo Lambda](https://www.cargo-lambda.info/) to package the code, and `template.yml`.
 
-1. Run the following command replacing `<region>` with the AWS region you want to deploy in and provide parameter overrides as desired:
+##### Stack Parameters
+
+The following parameters are available when deploying the connector as part of a CloudFormation stack. An example of setting these parameters is included in step 4 of the [SAM deployment steps](#sam-deployment-steps).
+
+| Parameter     | Description | Default Value |
+|---------------|-------------|---------------|
+| `DatabaseName`  | The name of the database to use for ingestion. | `influxdb-line-protocol` |
```

Comment:
The `Enable-` parameters are left out, as we actually don't want users to be that informed about them.